packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js

```diff
@@ -80,33 +84,33 @@ class DurationEdit extends React.Component {
 
   render() {
     return (
-      <StyledDurationEdit>
-        <input
-          type="number"
-          step={1}
-          onChange={() => {}} // Empty onChange function to prevent React internal error
-          className="form-control"
-          value={this.state.hours}
-          onInput={this.handleHourChange}
-          onKeyPress={this.preventNonNumeric}
-          disabled={this.props.readOnly}
-          pattern="\d+"
-          min={0}
-        />
+      <StyledDurationEditWrapper>
+        <StyledEditableWrapper>
+          <StyledDurationEdit
```

Comment:
Would it be simpler to write the styling for DurationEdit in one instead of two wrappers? We would then have StyledEditableWrappers and then StyledDurationEdit without the StyledDurationEditWrapper.

Reply:
Good point to reason about. It would be simpler but I could not achieve what I have intended to do. 
- StyledDurationEditWrapper is responsible for the horizontal layout and unique for DurationEdit. 
- StyledEditableWrapper is very generic and used in almost all editable fields. It is responsible to place buttons next to input fields and to emphasize a strong connection.  
- StyledDurationEdit is generic and used in a similar way in almost all editable fields. Only an additional attribute is defined. It is responsible to harmonize and to reset input field declarations.
It would be possible to remove one wrapper if only one Input field must have been wrapped.